lib/alerts/components/AlertEditor.js

```diff
@@ -1,29 +1,29 @@
 // @flow
 
-import Icon from '@conveyal/woonerf/components/icon'
-import moment from 'moment'
+import * as activeAlertActions from '../actions/activeAlert'
+import * as alertActions from '../actions/alerts'
+
+import type {Alert, Feed, GtfsRoute, GtfsStop, Project} from '../../types'
+import {Button, ButtonToolbar, Col, ControlLabel, FormControl, FormGroup, Grid, Row} from 'react-bootstrap'
+import {CAUSES, EFFECTS, isNew} from '../util'
 import React, {Component} from 'react'
-import {Grid, Row, Col, ButtonToolbar, Button, FormControl, ControlLabel, FormGroup} from 'react-bootstrap'
-import DateTimeField from 'react-bootstrap-datetimepicker'
-import {browserHistory} from 'react-router'
-import Toggle from 'react-toggle'
 
 import AffectedServices from './AffectedServices'
-import * as alertActions from '../actions/alerts'
-import * as activeAlertActions from '../actions/activeAlert'
+import type {Props as ContainerProps} from '../containers/ActiveAlertEditor'
+import DateTimeField from 'react-bootstrap-datetimepicker'
```

Comment:
Also, just a reminder to probably turn off your IDE's auto-reformatting of files. Generally we like to limit changes in PRs to the affected lines of code (rather than modifying a bunch of stuff wholesale).

Reply:
I had a "sort imports" plugin from ITMS running amok.... it has been disabled.  Just so you know I wasn't randomly moving lines. I should have double checked.

Reply:
No worries at all